server/routes/changeLocation/handlers/changeLocation.ts

```diff
@@ -11,15 +11,17 @@ export default class ChangeLocationRoutes {
       const prisonCaseloadFromNomis = await this.userService.getPrisonUserCaseloads(user)
       const caseload = prisonCaseloadFromNomis.map(c => ({ value: c.caseLoadId, text: c.description }))
       const checked = req.session.caseloadsSelected
```

Comment:
Change location and cancel link are not requested as part of the ticket's AC's. However the change location and cancel link need to be updated as now they serve 2 different views and following their execution, the user needs to to be returned to their original view.